app/routers/system.py

```diff
@@ -125,5 +131,7 @@ def reboot_system():
     summary="Shuts the system down",
     dependencies=[Depends(JWTBearer())],
 )
-def reboot_system():
-    return HTTPException(status.HTTP_501_NOT_IMPLEMENTED)
+async def reboot_system():
```

Comment:
Sorry, had a copy & paste error here :). The path function should be more like `def shutdown():` or similar.